db/model/Redirect.ts

```diff
@@ -16,6 +16,29 @@ export async function getRedirects(
     )
 }
 
+/**
+ * Takes a list of redirects and returns a map of source -> target
+ * such that if you have [A -> B, B -> C], the map will be { A -> C, B -> C }
```

Comment:
I thought there were no redirect chains in the DB. We have a validation that prevents adding new chains, and if there are any existing, it would be better if we resolved/changed them in the DB instead of always resolving them here.

Reply:
True! I had some leftover testing data in my redirects table it seems. Not that it matters now that I've removed the explorer validation 🙂